src/index.js

```diff
@@ -1,37 +1,57 @@
 import 'bootstrap/dist/css/bootstrap.css'
 
 import { Container, Navbar, NavbarBrand } from 'reactstrap'
-import { css } from 'emotion'
+import { css, cx } from 'react-emotion'
 import { faCompass } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import { GRAY_900 } from './assets/colors'
+import { GRAY_900, BLUE_GRAY_900 } from './assets/colors'
 import FeedbackButton from './components/feedback-button'
 import MortgageInvestmentCompare from './components/mortgage-investment-compare/index'
 
-function App () {
-  // TODO 018.12.09: Include an intro paragraph
-  return (
-    <div
-      className={css`
-        color: ${GRAY_900};
-        font-family: 'Noto Sans', sans-serif;
-      `}
-    >
-      <Navbar className='border-bottom sticky-top' color='white' light>
-        <NavbarBrand href='/'>
-          <FontAwesomeIcon icon={faCompass} className='mr-2 text-success' />
-          MoneyCompass
-        </NavbarBrand>
-      </Navbar>
-      <Container fluid className='pt-4 px-5'>
-        <MortgageInvestmentCompare />
-        <FeedbackButton />
-      </Container>
-    </div>
-  )
+const footer = cx(
+  css(`background-color: ${BLUE_GRAY_900};`),
+  'text-white',
+  'd-flex',
+  'flex-column',
+  'align-items-center',
+  'mt-5',
+  'p-4'
+)
+
+class App extends React.Component {
+  render () {
+    // TODO 018.12.09: Include an intro paragraph
+    return (
+      <div
+        className={css`
+          color: ${GRAY_900};
+          font-family: 'Noto Sans', sans-serif;
+        `}
+      >
+        <Navbar className='border-bottom sticky-top' color='white' light>
+          <NavbarBrand href='/'>
+            <FontAwesomeIcon icon={faCompass} className='mr-2 text-success' />
+            MoneyCompass
+          </NavbarBrand>
+        </Navbar>
+        <Container fluid className='pt-4 px-5'>
+          <MortgageInvestmentCompare />
+          <FeedbackButton />
+        </Container>
+        <footer className={footer}>
```

Comment:
Are there no namespace conflicts here?

Reply:
footer is also an html element

Reply:
It's neat that React can intelligently pick between custom and standard tags

Reply:
yeah never really thought about it. I wonder what would happen if I created a styled component `footer`. Feel like it would just overwrite it